mailpoet/lib/Form/Block/Select.php

```diff
@@ -51,7 +51,7 @@ class="mailpoet_select"
       if (!empty($block['params']['required'])) {
         $label .= ' *';
       }
-      $html .= '<option value="" disabled selected hidden>' . $label . '</option>';
+      $html .= '<option value="" disabled selected hidden>' . $this->wp->escAttr($label) . '</option>';
```

Comment:
I think that for this case, we should use escHtml as this is not printed inside an attribute.